nix/libvirtd.nix

```diff
@@ -47,6 +47,15 @@ in
       '';
     };
 
+  options = {
+    deployment.libvirtd.URI = mkOption {
+      type = types.str;
```

Comment:
Could this benefit from types.enum?

Reply:
This may or may not reveal how little I know about libvirt :-)

Reply:
> Could this benefit from types.enum?

Probably not, this option can be an arbitrary URI string: https://libvirt.org/uri.html.

It would be nice to have types.URL and types.URI though. :)